src/apps.rs

```diff
@@ -17,24 +18,29 @@ pub fn create(url: &Url, app: &AppId, data: serde_json::Value) {
     let res = client.post(&url)
         .header(reqwest::header::CONTENT_TYPE, "application/json")
         .body(body.to_string())
-        .send();
+        .send().with_context(|| format!("Can't create app "))?;
```

Comment:
`./target/debug/cli -u ht:google.com create app app1`
```
Error: Can't create app 

Caused by:
    0: builder error for url (ht:google.comapi/v1/apps): URL scheme is not allowed
    1: URL scheme is not allowed
```

In my opinion, enough context is provided by reqwest library. Do we need to still enrich this error message? 

As a bonus, we can create a custom error type example `HttpRequestError`, it may help us later while writing test cases.

Reply:
I initially thought the only error message we would see was "Can't create app". Sorry for the confusion. I think it is good as it is.